jetstream/cli.py

```diff
@@ -217,11 +214,21 @@ def validate_config(path):
         click.echo(f"Validate {file}", err=False)
 
         spec = AnalysisSpec.from_dict(toml.load(file))
-        spec.resolve(dummy_experiment)
+        config = spec.resolve(dummy_experiment)
 
         # check if there is an experiment with a matching slug in Experimenter
         slug = os.path.splitext(os.path.basename(file))[0]
         if collection.with_slug(slug).experiments == []:
             click.echo(f"No experiment with slug {slug} in Experimenter.", err=False)
```

Comment:
I think we should actually fail here (`sys.exit(1)`) so that users get feedback that they might have typoed something. They can merge anyway if they're sure they're right. 😬

Reply:
We might run into issues where experiments disappear from Experimenter and then validation fails for configs unrelated to the one the user was working on. I don't think we have to worry about this too much for now though

Reply:
Yeah; if that gets annoying it shouldn't be too much git trickery to only have CI consider the files that changed in a push.